ygnmi/ygnmi_test.go

```diff
@@ -2361,6 +2361,134 @@ func TestBatchGet(t *testing.T) {
 	}
 }
 
+func TestBatchWatch(t *testing.T) {
+	fakeGNMI, c := newClient(t)
+	aLeafStatePath := testutil.GNMIPath(t, "/remote-container/state/a-leaf")
+	twoPath := testutil.GNMIPath(t, "/parent/child/state/two")
+	aLeafSubPath := testutil.GNMIPath(t, "/remote-container/*/a-leaf")
+
+	tests := []struct {
+		desc                 string
+		stub                 func(s *testutil.Stubber)
+		config               bool
+		paths                []ygnmi.PathStruct
+		wantSubscriptionPath []*gpb.Path
+		wantVal              *ygnmi.Value[*exampleoc.Root]
+		wantErr              string
+	}{{
+		desc: "predicate true",
```

Comment:
Add an overwrite test with multiple notifications? I'm thinking can add a false->true and a true->false test, and in the latter you're checking that the false notification is ignored.

Reply:
added test where pred is false, then true, then false